### ISSUE TITLE
Can't stop server

### ISSUE THREAD

**Issue by dgrant** [dgrant/eyefiserver2#10]
eyefiserver2 can't be stopped using init.d script (needs to be -9'd). This only seems to apply after a photo has been uploaded. If not uploading has occurred then it will shut down.

**Comment by aflat** [comment]
If I change line 181 from
````python
os.kill(pid, SIGTERM)
````
to
````python
os.kill(pid, signal.SIGQUIT)
````
It shuts down fine, and doesn't core dump either. Tested on Mint17, and Fedora 19.

**Comment by aflat** [comment]
Looks like that change doesn't allow the server to stop after I upload a pic, so back to the drawing board.

**Comment by dgrant** [comment]
Thanks for trying. :-)

**Comment by aflat** [comment]
Ok, not nearly as nice, it's just killing it, but that's what I have to do anyways. 
````python
os.kill(pid, 9)
````
works, even after I upload a picture. It does clean up the pidfile too.

**Comment by ryantm** [comment]
I can confirm that when running as an instance and doing ctrl-c after uploading a picture, it doesn't kill it.

**Comment by Sklanfurt** [comment]
Got the script running but my computer wont shut down after it started. Have to kill it everytime. Somebody got a solution for this?

**Comment by ryantm** [comment]
@Sklanfurt My solution was to write my own server. The code is at https://github.com/ryantm/heyefi

**Comment by dgrant** [comment]
@ryantm in Haskell! :raised_hands:

**Comment by dgrant** [comment]
Yeah this bug sucks, I can't even reboot my NAS, I always have to ssh in to it and stop it before doing a reboot otherwise it will never shut down. I spend some time trying to fix it a while back and failed.
